crates/sui-package-resolver/src/lib.rs

```diff
@@ -93,6 +96,32 @@ pub struct Package {
 
 type Linkage = BTreeMap<AccountAddress, AccountAddress>;
 
+#[derive(Clone, Debug)]
+pub enum CleverError {
+    CompleteError {
```

Comment:
nit (optional): avoiding name redundancy

```suggestion
    Complete {
```